### ISSUE TITLE
Return to single send/wait

### ISSUE THREAD

**Issue by tjguk** [tjguk/networkzero#29]
I previously implemented the means of sending to multiple addresses (supported under the covers in ZeroMQ by connecting to multiple endpoints). However...

* This was largely "because I can" 
* The semantics are those of the REQ/REP pair (ie round robin) which are probably not the most intuitive (prob. send to all).
* There's scope for send-to-any / send-to-all but they should be more explicity and, in particular, clearer about how replies are handled.

In short, I propose to remove this functionality from the main development branch and play with alternative implementations in a feature branch

**Comment by tjguk** [comment]
Now complete and merged into master
